docs/guides/virtualization/cloud-init/02_first_contact.md

```diff
@@ -0,0 +1,151 @@
+---
+Title: II. First Contact: Simple Bootstrapping
+author: Wale Soyinka
+contributors:
+tags:
+  - cloud-init
+  - cloud
+  - automation
+---
+
+
+# **II. First Contact: Simple Bootstrapping with the Rocky Linux 10 QCOW2 Image**
+
```

Comment:
Again, this level 1 heading should be deleted and just allow the Title meta to populate it. If you like, you can add a level 2 heading that has "with the Rocky Linux 10 QCOW image" before the content starts OR add that to the Title meta. Again, the additional bold facing of the headings throughout is not needed as each heading will add its own emphasis.